### PR TITLE
Add netstandard20 target

### PR DESCRIPTION
**Pull request by hanabi1224**
Address https://github.com/multiformats/cs-multihash/issues/25

Also add [sourcelink](https://github.com/dotnet/sourcelink) and [symbol package](https://github.com/NuGet/Home/wiki/NuGet-Package-Debugging-&-Symbols-Improvements)

### CODE REVIEW

**Comment by tabrath** [comment]
Great work! Love the sourcelink inclusion. Also very nice to see all those lines deleted from the appveyor config, it just looks like you just missed a couple of parameters to generate the coverage file.

File: appveyor.yml [hunk 1]
```diff
@@ -52,33 +49,20 @@ build_script:
...
 test: off
 
 test_script:
-  - dotnet test --configuration Debug --framework netcoreapp2.0 --logger trx;logfilename=TestResult.xml --results-directory "%APPVEYOR_BUILD_FOLDER%\test\Multiformats.Hash.Tests\TestResults" /p:CollectCoverage=true /p:CoverletOutputFormat=opencover "%APPVEYOR_BUILD_FOLDER%\test\Multiformats.Hash.Tests\Multiformats.Hash.Tests.csproj"
-
-after_test:
-  - ps: |
-        $wc = New-Object 'System.Net.WebClient'
-        $wc.UploadFile("https://ci.appveyor.com/api/testresults/xunit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\test\Multiformats.Hash.Tests\TestResults\TestResult.xml))
+  - dotnet test -c %CONFIGURATION% -f netcoreapp2.0 /p:CollectCoverage=true "%APPVEYOR_BUILD_FOLDER%\test\Multiformats.Hash.Tests"
```
Comment (tabrath) [inline appveyor.yml]:
```suggestion
  - dotnet test -c %CONFIGURATION% -f netcoreapp2.0 /p:CollectCoverage=true /p:CoverletOutputFormat=opencover "%APPVEYOR_BUILD_FOLDER%\test\Multiformats.Hash.Tests"
```

File: test/Multiformats.Hash.Tests/Multiformats.Hash.Tests.csproj [hunk 2]
```diff
@@ -20,22 +20,30 @@
     <DefineConstants>$(DefineConstants)</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- https://github.com/tonerdo/coverlet -->
+    <!-- Do not collect by default -->
+    <CollectCoverage>false</CollectCoverage>
```
Comment (hanabi1224) [inline test/Multiformats.Hash.Tests/Multiformats.Hash.Tests.csproj]: Here are all the properties for codecov

**Comment by hanabi1224** [comment]
@tabrath If you look at appveyor ci log, it looks like it uploads codedov properly 

image

File: test/Multiformats.Hash.Tests/Multiformats.Hash.Tests.csproj (hunk 2 above)
Comment (tabrath) [reply test/Multiformats.Hash.Tests/Multiformats.Hash.Tests.csproj]: My bad. I was confused because one check from codecov fails. Looks good to me.

**Comment by tabrath** [comment]
Nice work. Fixes #25

**Pull request merged**